source/install/transport-encryption/config-cluster.rst

```diff
@@ -0,0 +1,231 @@
+=============================================
+Configuring Cluster Transport Encryption (E20)
+=============================================
+
+Mattermost is able to encrypt the messages sent within the cluster of a deployment
+using SSH tunneling. The guide walks through the deployment of this solution on
+Ubuntu 18.04, but it can be adapted for any Linux operating system.
+
+While this document only describes the configuration of a three-node cluster, it is
+by no means limited to that amount.
+
+## Prerequisites
+- SSH port whitelisted between each node of the deployment
+- ufw / iptables is active on each node
+- Access to the root / sudo user of each node for configuration
+- A configured Mattermost cluster
+- Mattermost running with a dedicated service user
+- Mattermost service is stopped on each cluster node
+
+**Note:** A support on the application level is currently in development and will
```

Comment:
```suggestion
**Note:** Support on the application level is currently in development and will
```
Not sure if that change works - I may have misunderstood the sentence. From what I can see, it refers to support offered by us on the application level?